history/management/commands/run_bot.py

```diff
@@ -185,10 +218,47 @@ def leaderboard(message,gamename):
 
             stats_by_user = sorted(stats_by_user.items(), key=lambda x: -1 * x[1]['elo'])
 
+            season_str = "All time" if active_season is None else active_season
             stats_str = "\n ".join([  " * {}({}): {}/{} ({}%)".format(stats[1]['name'],stats[1]['elo'],stats[1]['wins'],stats[1]['losses'],stats[1]['win_pct'])  for stats in stats_by_user ])
-            stats_str = "Leaderboard for {}: \n\n{}\n{}".format(gamename, stats_str,_get_elo_graph(gamename))
+            stats_str = "{} leaderboard for {}: \n\n{}\n{}".format(season_str, gamename, stats_str,_get_elo_graph(gamename,range_start_date))
             message.send(stats_str)
 
+        @listen_to('^season (.*)',re.IGNORECASE)
+        @listen_to('^gamebot season (.*)',re.IGNORECASE)
+        @listen_to('^gb season (.*)',re.IGNORECASE)
+        def season(message,gamename):
+            #input sanitization
+            gamename = gamename.strip()
+
+            #close current season
+            active_season, start_on = get_active_season(gamename,True)
+
+            #msg back to users
+            msg_str = "{} is active. \nUse `gamebot end season {}` to end this season.".format(active_season, gamename)
+            message.send(msg_str)
+
+
+        @listen_to('^end season (.*)',re.IGNORECASE)
+        @listen_to('^gamebot end season (.*)',re.IGNORECASE)
+        @listen_to('^gb end season (.*)',re.IGNORECASE)
```

Comment:
I feel like we should have some sort of lock on this command. I could see it being abused pretty easily. Any ideas on that front? Not necessary?